src/main/kotlin/com/buildit/bookit/auth/OpenidSigningKeyResolver.kt

```diff
@@ -30,9 +32,44 @@ internal class OpenidSigningKeyResolver : SigningKeyResolverAdapter() {
      * (There are about three keys which are rotated about everyday.)
      */
     private fun loadPublicKey(soughtKid: String): PublicKey? {
-        // TODO: cache content to file to prevent access internet every time.
+        if (!::jwkConfig.isInitialized) {
+            jwkConfig = fetchJwkConfig()
+        }
```

Comment:
dunno whether lateinit is better or just init in the declaration (spring context).... given that both ultimately will get initialized during the first request, i think it's better in the declaration.. will change